bcda/service/config.go

```diff
@@ -53,10 +53,11 @@ type RunoutConfig struct {
 
 type ACOConfig struct {
 	Model              string
-	Pattern            string `conf:"name_pattern"`
-	PerfYearTransition string `conf:"performance_year_transition"`
-	LookbackYears      int    `conf:"lookback_period"`
-	Disabled           bool   `conf:"disabled" conf_default:"false"`
+	Pattern            string   `conf:"name_pattern"`
+	PerfYearTransition string   `conf:"performance_year_transition"`
+	LookbackYears      int      `conf:"lookback_period"`
+	Disabled           bool     `conf:"disabled" conf_default:"false"`
+	Data               []string `conf:"data"`
```

Comment:
Since it looks like every current model has the `"adjudicated"` trait, would it make sense to set this as the default in our config so we have to less to set in the config file(s) itself.

Reply:
yeah that makes sense, is the following the correct format for `conf_default`?
```
	Data               []string `conf:"data" conf_default:"[1]string{\"adjudicated\"}"`
```

Reply:
Im actually not sure myself when it comes to a list, ill double check as well.

Reply:
@SMLuthi I took a deep look into this and I don't see an easy path for supporting default values for complex (slices, maps, etc) fields in field tags.

If you want to pair on this some more and try to figure out what changes we can make to `conf/config.go` to support this, I'm open.

Reply:
@ericbuckley Like we talked about yesterday. I think this is outside the scope of your change. We can note this down as a ticket for future improvement but I'd rather you not get bogged down trying to make a change to the configuration package!